developers.diem.com/README.md

```diff
@@ -62,7 +46,3 @@ Get the latest updates to our project by signing up to our [newsletter](https://
 ## License
 
 Diem Core is licensed as [Apache 2.0](https://github.com/diem/diem/blob/master/LICENSE)
-
-## Docs Directory
```

Comment:
@oriooctopus Just curious why you deleted this. Was there duplicative information somewhere else?

Reply:
@JoelMarcey I have re-added most of the removed links, but there are still a few that are missing because we don't know what to change them to. If you don't want them to be removed, would you mind commenting on https://docs.google.com/spreadsheets/d/1Lo6uWkKz5a7Cs8_7sfSRvlUXD3J3G07iV5GQexF2mO4/edit#gid=0 with the updated link?